examples/src/main/scala/com/basho/riak/spark/examples/dataframes/SimpleScalaRiakTSDataframesExample.scala

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2015 Basho Technologies, Inc.
+ *
+ * This file is provided to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain
+ * a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.basho.riak.spark.examples.dataframes
+
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.SaveMode
+import org.apache.spark.sql.riak.RiakSQLContext
+
+/**
+ * Example shows how Spark DataFrames can be used with Riak TS
+ * * CREATE TABLE ts_weather_demo
+ *  (
+ *     weather      varchar not null,
+ *     family       varchar not null,
+ *     time         timestamp not null,
+ *     temperature  double,
+ *     humidity     double,
+ *     pressure     double,
+ *     PRIMARY KEY (
+ *         (weather, family, quantum(time, 1, 'h')), weather, family, time
+ *     )
+ *  )
+ */
+object SimpleScalaRiakTSDataframesExample {
+  private val tableName = "ts_weather_demo"
+
+  def main(args: Array[String]) {
+    val sparkConf = new SparkConf().setAppName("Riak TS Spark Dataframes Example")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+
+    val sc = new SparkContext(sparkConf)
+
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+    import sqlContext.implicits._
+
+    // Load test data from json file
+    println("---------------------------------- input data -----------------------------------")
+    val inputDF = sqlContext.read.json("src/main/resources/ts_test_data.json")
```

Comment:
IMO it might be more demonstrably it you embed JSON strings, instead of loading them from file